apps/server/src/modules/authentication/strategy/oauth2.strategy.ts

```diff
@@ -31,13 +32,16 @@ export class Oauth2Strategy extends PassportStrategy(Strategy, 'oauth2') {
 			throw new UnauthorizedException('no account found');
 		}
 
+		if (account.deactivatedAt != null && account.deactivatedAt.getTime() <= Date.now()) {
+			throw new UserAccountDeactivatedLoggableException();
+		}
+
 		const currentUser: OauthCurrentUser = CurrentUserMapper.mapToOauthCurrentUser(
 			account.id,
 			user,
 			systemId,
 			tokenDto.idToken
 		);
-
 		return currentUser;
 	}
 }
```

Comment:
what about other login strategies?

Reply:
other strategies use a method loadAccouny from authorization.service where this code is also implemented